spanner/opentelemetry_traces/pom.xml

```diff
@@ -49,10 +49,18 @@
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-spanner</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
```

Comment:
This dependency is already part of Spanner Java Client library. Why do we need to add this again here ?

Reply:
As discussed offline with you @surbhigarg92 that we should only declare dependencies that needs to be added by customers, not making any changes to explicitly declare the dependencies.